msrest/serialization.py

```diff
@@ -1486,6 +1486,25 @@ def _classify_target(self, target, data):
             pass  # Target is not a Model, no classify
         return target, target.__class__.__name__
 
+    def failsafe_deserialize(self, target_obj, response_data, content_type=None):
```

Comment:
made the signature the same as `__call__`, since I wanted calling `failsafe_deserialize` the same as calling deserializer()